lib/rubocop/cop/lint/format_parameter_mismatch.rb

```diff
@@ -80,6 +80,7 @@ def offending_node?(node)
           num_of_format_args, num_of_expected_fields = count_matches(node)
 
           return false if num_of_format_args == :unknown
+          return false if num_of_expected_fields&.zero? && node.child_nodes.first.dstr_type?
```

Comment:
Is the safe navigation operator required?

```suggestion
          return false if num_of_expected_fields.zero? && node.child_nodes.first.dstr_type?
```

Reply:
Ah, initially it was necessary because the order was reversed with L82, so :unknown came in, but now it is not necessary. Thank you very much.